frontend/src/components/ViewTrips/trips-container.js

```diff
@@ -65,17 +66,21 @@ class TripsContainer extends React.Component {
         .where(DB.TRIPS_COLLABORATORS, 'array-contains', curUserUid)
         .orderBy(DB.TRIPS_UPDATE_TIMESTAMP, 'desc')
         .onSnapshot(querySnapshot => {
-          const tripsContainer = querySnapshot.docs.map(doc =>
+          const tripsContainer = querySnapshot.docs.map((doc, idx) =>
               ( <Trip
                   tripData={doc.data()}
                   tripId={doc.id}
                   handleEditTrip={this.props.handleEditTrip}
+                  eventKey={String(idx)}
                   key={doc.id}
                 />
               )
           );
 
-          this.setState({ trips: tripsContainer });
+          this.setState({
+                          trips: tripsContainer,
+                          firstTripId: querySnapshot.docs[0].id,
```

Comment:
why is firstTripId useful? I don't see it being used anywhere else.

Reply:
Whoops I took this out in #107 but forgot to take it out here. Fixed.